blog-service/2024-04-01-alerts-monitors.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Muting Schedules for Alert Groups
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - alerts
+  - monitors
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+You can now optionally apply an alert Muting Schedule to your alert group. This saves time by allowing you to select an alert group (a bundled set of alerts) to associate with your Muting Schedule rather than selecting individual monitors. [Learn more](/docs/alerts/monitors/muting-schedules/#set-a-muting-schedule-for-an-alert-group).
```

Comment:
```suggestion
You can now optionally apply a Muting Schedule to your alert group. This saves time by allowing you to select an alert group (a bundled set of alerts) to associate with your Muting Schedule rather than selecting individual monitors. [Learn more](/docs/alerts/monitors/muting-schedules/#set-a-muting-schedule-for-an-alert-group).
```

Reply:
one minor suggestion

Reply:
The release note looks good with @gchairuangsang's suggestion.